### ISSUE TITLE
Support for label_id

### ISSUE THREAD

**Issue by seth586** [basnijholt/adaptive-lighting#968]
Home Assistant 2024.4 supports [labels](https://www.home-assistant.io/blog/2024/04/03/release-20244/#labels-tag-everything-any-way-you-want)

It would be nice to use `label_id` targets instead of the more explicit `entity_id`'s

**Comment by myklebosten** [comment]
Yes, this would be really nice! Came here to post the same question actually :)

As AL already expands light groups and send adapt commands to individual lights, this seem to me like it should be possible to implement?

**Comment by Tjomsen** [comment]
second this.

**Comment by adrianipopescu** [comment]
+1 on this
